README.md

```diff
@@ -53,3 +54,23 @@ If you plan to develop the code, please refer to the Wiki entries on <a href="ht
 
 We strongly encourage that code developments are properly documented.  Please refer to the <a href="https://github.com/EDmodel/ED2/wiki/ED2-Documentation-with-Doxygen">Doxygen</a> instructions, and especially the <a href="https://github.com/EDmodel/ED2/wiki/ED2-Documentation-with-Doxygen#doxygit"> Doxygen and Git commits</a> section, so additional documentation can be automatically generated from the source code comments.
 
+## <a name="docker"> Using Docker</a>
```

Comment:
Thanks for adding this! By the way, we can also include additional documentation (if needed) in the Wiki.